reservations/templates/reservations/reservation_details.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'rooms/base.html' %}
+
+{% block content %}
+
+<head>
+    <meta charset="UTF-8">
+    <title>{{reservation}} Details</title>
+</head>
+
+<body>
```

Comment:
The entire `<head` block and the two `<body>` tags can be removed when you extend the base